wgpu-core/src/device/resource.rs

```diff
@@ -3568,6 +3575,10 @@ impl<A: HalApi> Device<A> {
             let _ = texture.destroy();
         }
     }
+
+    pub(crate) fn get_usage_scope<'a>(&'a self) -> UsageScope<'a, A> {
```

Comment:
`get_` is not idiomatic rust and it's not even a getter as this creates a new object.
Rust is also perfectly fine with implicit lifetimes here :)
```suggestion
    pub(crate) fn new_usage_scope(&self) -> UsageScope<'_, A> {
```

Reply:
> this creates a new object

well, sometimes ...

Reply:
well the _scope_ is always new 😄